front/gatsby/src/html.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
```

Comment:
Gatsby generates the public folder on each build, so the `index.html` file always get reset to their default configuratoin, modifying this html.js file lets us persist some things. For tis case it was for the semantic ui css CDN, but in the future if we have any more CDNs this would be the place to add them. https://www.gatsbyjs.org/docs/custom-html/